src/ui/components/atoms/burger-button/BurgerButton.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import type { StoryObj } from '@storybook/react';
+import BurgerButton from './BurgerButton';
+import IconBurger from '../../../../assets/icons/button-burguer.svg?raw';
+
+const meta = {
+  title: 'ui/components/atoms/burger-button',
+  component: BurgerButton,
+  argTypes: {
+    onClick: {
+      action: 'click',
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Primary: Story = {
+  args: {
+    icon: IconBurger,
+    variant: 'main',
+    onClick: () => {
+      console.log('Button clicked');
+    },
```

Comment:
This component is pretty simple you need to add more variants example change the color!